src/System.Runtime.Numerics/tests/BigInteger/ctor.cs

```diff
@@ -924,7 +924,7 @@ public static void RunCtorByteArrayTests()
             // ctor(byte[]): array is large
             for (int i = 0; i < s_samples; i++)
             {
-                tempByteArray = new byte[s_random.Next(16384, 2097152)];
+                tempByteArray = new byte[s_random.Next(16384, 262144)];
```

Comment:
Is anyone familiar with the default stack size and at what number of digits this will blow up? My first impression is that this is a regression for (sufficiently) large numbers, and thus needs to be addressed before we can take the change in.